source/documentors/concepts/content_types.rst

```diff
@@ -1,49 +1,342 @@
-Documentation Content Types
-###########################
+Open edX Diataxis Guide
+#######################
 
-To keep documentation streamlined and make it easy for users to find answers, we make each topic one of four types:
+.. contents:: 
+  :local:
+  :depth: 1
+
+Diataxis Definition
+*******************
+
+The Diataxis framework is an approach to quality in technical documentation and creates a systematic organization. Diataxis identifies four modes of documentation:
+
+* `Quickstart`_
+* `How-to`_
+* `Reference`_
+* `Concept`_
+
+Using diataxis, documentation can be organized to meet diverse user needs, making it easier to find relevant information based on the user’s situation or goal. This structure enhances usability and comprehension, allowing users to navigate documentation more effectively.
+
+For each type of document, you will find the explanation, the technical guidelines, and a matrix of criteria that will allow you to identify whether or not a document complies with the parameters of the diataxis used in the Open edx repositories. 
 
 .. _Quickstart:
 
-****************
-**Quickstart** 
-****************
+Quickstart
+**********
+
+A Quickstart is a lesson that guides the reader through **practical steps** to complete a stated goal. It is **learning-oriented**. In other words, it is a lesson and a learning experience. For example, creating a virtual course can be a Quickstart because it’s a practical skill. A Quickstart may comprise the contents of multiple How-To guides
```

Comment:
```suggestion
A Quickstart is a lesson that guides the reader through **practical steps** to complete a stated goal. It is **learning-oriented**. In other words, it is a lesson and a learning experience. For example, creating a virtual course can be a Quickstart because it’s a practical skill. A Quickstart may comprise the contents of multiple How-To guides.
```